src/AddCard/constants/locations.ts

```diff
@@ -0,0 +1,7 @@
+export const locations = {
+  BASE_URL: "/react-payments",
+  ADD_CARD: { pathname: "" },
+  ADD_CARD_COMPLETE: {
+    pathname: "complete",
+  },
+};
```

Comment:
👍 👍 👍

Reply:
아래처럼 작성하지 않고 1depth를 더 두신 이유가 있을까요?

```suggestion
export const locations = {
  BASE_URL: "/react-payments",
  ADD_CARD: "",
  ADD_CARD_COMPLETE:  "complete",
};
```

Reply:
아, 이것은, url과 state를 같이 실어서 보관을 할수 있더라구요? 그래서 뭔가 initialState 같은것도 값을 주려고 생각은 해두었는데, 쓸모 없는 코드는 지우는게 좋으니까요. 지웠어용!